packages/es-components/src/components/containers/table/Table.tsx

```diff
@@ -45,8 +45,8 @@ const TableBase = styled.table<TableBaseProps>`
   }
 
   th {
-    font-family: 'Source Sans Pro', 'Segoe UI', Segoe, Calibri, Tahoma,
-      sans-serif;
+    font-family:
+      'Source Sans Pro', 'Segoe UI', Segoe, Calibri, Tahoma, sans-serif;
```

Comment:
This doesn't need to be updated with this PR, but the font should come from the theme, rather than being hard-coded.